portal-impl/impl/src/java/org/sakaiproject/portal/charon/QuickLinkComparator.java

```diff
@@ -0,0 +1,18 @@
+package org.sakaiproject.portal.charon;
+
+import java.util.Comparator;
+import java.util.Map;
+
+/**
+ * Created by neelam on 24/07/2015.
+ * Comparator to Sort the quick links
+ */
+public class QuickLinkComparator implements Comparator {
+
+    public int compare(Object object1, Object object2) {
+        Map<String, String> quickLink1 = (Map<String, String>) object1;
+        Map<String, String> quickLink2 = (Map<String, String>) object2;
+        return quickLink1.get("name").compareTo(quickLink2.get("name"));
```

Comment:
Will NPE if quicklink1 doesn't have a name which from the building of the list looks to be possible.